guides/source/action_cable_overview.md

```diff
@@ -454,7 +499,7 @@ method on `current_user`. (`appearance_channel.rb`)
 4. **Client** recognizes that a subscription has been established and calls
 `connected` (`appearance.coffee`) which in turn calls `@install` and `@appear`.
 `@appear` calls `AppearanceChannel#appear(data)` on the server, and supplies a
-data hash of `{ appearing_on: $("main").data("appearing-on") }`. This is
+data hash of `{ appearing_on: main.getAttribute("data-appearing-on") }`. This is
 possible because the server-side channel instance automatically exposes all
 public methods declared on the class (minus the callbacks), so that these can be
 reached as remote procedure calls via a subscription's `perform` method.
```

Comment:
This "Client-Server Interaction" list could use more freshening up. Many of the items still refer to the old CoffeeScript-based syntax.

Reply:
There was also a snippet below the list that was still coffee syntax. I think that's all of them now, thanks!